src/scheduled_tasks.rs

```diff
@@ -147,6 +156,45 @@ fn clear_old_activities(conn: &mut PgConnection) {
   }
 }
 
+/// overwrite posts and comments 30d after deletion
+fn overwrite_deleted_posts_and_comments(conn: &mut PgConnection) {
+  info!("Overwriting deleted posts...");
+  match diesel::update(
+    post::table
+      .filter(post::deleted.eq(true))
+      .filter(post::updated.lt(now - 30.days()))
+      .filter(post::body.ne("Deleted")),
+  )
+  .set(post::body.eq("Deleted"))
```

Comment:
Should we not harmonize this with the text used for example in https://github.com/LemmyNet/lemmy/blob/main/crates/db_schema/src/impls/comment.rs#L34 (and fixing the typo)?

Reply:
Would also be good to localize it, see how its done for email

Reply:
I didn't tackle localization yet (in part because it saves the actual string to the database, so I'm not sure if we should localize based on the author, the instance, or save the token and then localize it on the way back out for display purposes?

but I did tackle making this standard with what is already there in comment & post, namely having it set the text to 
```rust
    let perma_deleted = "*Permanently Deleted*";
```